simvue/api/objects/run.py

```diff
@@ -351,3 +350,21 @@ def abort(self, reason: str) -> dict[str, typing.Any]:
             scenario=f"Abort of run '{self.id}'",
             response=_response,
         )
+
+    def on_reconnect(self, id_mapping: dict[str, str]):
+        online_alert_ids = []
```

Comment:
Please type this list as `mypy` will not be able to ascertain the type of it's elements from `id_mapping[id]`